src/libs/blackboard/interface_listener.cpp

```diff
@@ -114,12 +114,24 @@ BlackBoardInterfaceListener::bbil_name() const
 	return name_;
 }
 
-/** BlackBoard data changed notification.
+/** BlackBoard data refreshed notification.
  * This is called whenever the data in an interface that you registered for is
  * modified. This happens if a writer calls the Interface::write() method.
  * @param interface interface instance that you supplied to bbil_add_data_interface()
  */
```

Comment:
Please clarify that "modified" does not necessarily mean "changed"

Reply:
Removed confusing "modified" wording and clarified change vs. refresh. Please take another look.